packages/react-google-maps-api/src/utils/injectscript.ts

```diff
@@ -48,11 +48,11 @@ export const injectScript = ({ url, id }: InjectScriptArg): Promise<any> => {
     script.id = id
     script.async = true
     script.onerror = reject
-    script.setAttribute('state', 'loading')
+    script.setAttribute('data-state', 'loading')
```

Comment:
Do we actually need to set that attribute at this point? Since the only check for `ready` is done and `getAttribute` will return `null`, there is no need to tweak DOM more than necessary imo.

Reply:
Yeah, fair point